src/display/mapcanvas.cpp

```diff
@@ -1490,6 +1490,50 @@ void MapCanvas::drawRoomDoorName(const Room *sourceRoom, uint sourceDir, const R
     qint32 dX = srcX - tarX;
     qint32 dY = srcY - tarY;
 
+    // Look at door flags to code postfixes
+    DoorFlags sourceDoorFlags = getDoorFlags(sourceRoom->exit(sourceDir));
```

Comment:
Do we not care about the other two types of DF_NOBLOCK and DF_NOBREAK ?

Reply:
How often are these flags used independently? Can we use one flag for NOBLOCK and NOBREAK?